plotly/plotly/plotly.py

```diff
@@ -33,6 +33,7 @@
 from plotly.session import (sign_in, update_session_plot_options,
                             get_session_plot_options, get_session_credentials,
                             get_session_config)
+from plotly.grid_objs import grid_objs
```

Comment:
What do you mean by cyclic import? I'm honestly not too concerned about having it in there at all and just keeping it as a standard import, just wondering if folks didn't want that module to be relying on `grid_objs`